irr/include/irrMath.h

```diff
@@ -85,44 +76,44 @@ inline f32 degToRad(f32 degrees)
 /** Provided as it can be clearer to write degToRad(X) than DEGTORAD * X
 \param degrees The degrees value to convert to radians.
 */
-inline f64 degToRad(f64 degrees)
+inline constexpr f64 degToRad(f64 degrees)
 {
 	return DEGTORAD64 * degrees;
 }
 
-//! returns minimum of two values. Own implementation to get rid of the STL (VS6 problems)
+//! returns minimum of two values.
 template <class T>
 inline const T &min_(const T &a, const T &b)
 {
 	return a < b ? a : b;
 }
 
-//! returns minimum of three values. Own implementation to get rid of the STL (VS6 problems)
+//! returns minimum of three values.
 template <class T>
 inline const T &min_(const T &a, const T &b, const T &c)
 {
 	return a < b ? min_(a, c) : min_(b, c);
 }
 
-//! returns maximum of two values. Own implementation to get rid of the STL (VS6 problems)
+//! returns maximum of two values.
 template <class T>
 inline const T &max_(const T &a, const T &b)
 {
 	return a < b ? b : a;
 }
 
-//! returns maximum of three values. Own implementation to get rid of the STL (VS6 problems)
+//! returns maximum of three values.
 template <class T>
 inline const T &max_(const T &a, const T &b, const T &c)
 {
 	return a < b ? max_(b, c) : max_(a, c);
 }
 
-//! returns abs of two values. Own implementation to get rid of STL (VS6 problems)
+//! returns abs of two values.
 template <class T>
 inline T abs_(const T &a)
```

Comment:
i understand this use, is there many calls or can we remove it to use std::abs when needed ?

Reply:
there aren't many, but I'll leave the cleanup of this for someone else.